test/lib/fhirValidatorTest.ts

```diff
@@ -7,302 +7,60 @@ import sinonChai from 'sinon-chai';
-import questionnaireResponseExampleR4Training2 from './resources/r4/treatment-series-post-training-response-2-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4Training3 from './resources/r4/treatment-series-post-training-response-3-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4Training4 from './resources/r4/treatment-series-post-training-response-4-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4Training5 from './resources/r4/treatment-series-post-training-response-5-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4PreSession from './resources/r4/treatment-series-pre-session-response-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4PreTraining1 from './resources/r4/treatment-series-pre-training-response-1-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4PreTraining2 from './resources/r4/treatment-series-pre-training-response-2-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4PreTraining3 from './resources/r4/treatment-series-pre-training-response-3-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4PreTraining4 from './resources/r4/treatment-series-pre-training-response-4-en.questionnaireresponse.json';
-import questionnaireResponseExampleR4PreTraining5 from './resources/r4/treatment-series-pre-training-response-5-en.questionnaireresponse.json';
-
+delete r4QuestionnaireResponseExamples.QuestionnaireresponseExample;
+// includes 'Location'
+delete r4EncounterExamples.EncounterHomeExample;
 // includes 'Bundle', as do many other examples from the HL7 site not in the repo
-// import diagnosticReportExampleR4 from './resources/r4/diagnosticreport-example.json';
-import diagnosticReportExampleR4F201BrainCT from './resources/r4/diagnosticreport-example-f201-brainct.json';
-import diagnosticReportExampleR4DXA from './resources/r4/diagnosticreport-example-dxa.json';
-import diagnosticReportExampleR4UltraSound from './resources/r4/diagnosticreport-example-ultrasound.json';
-import diagnosticReportExampleR4PapSmear from './resources/r4/diagnosticreport-example-papsmear.json';
-import diagnosticReportExampleR4GingavalMass from './resources/r4/diagnosticreport-example-gingival-mass.json';
-
-import ResearchSubjectExampleR4 from './resources/r4/researchsubject-example.json';
-
-import fhirService from '../../src/services/fhirService';
-import { FHIR_VERSION_STU3, FHIR_VERSION_R4 } from '../../src/lib/models/fhir/helper';
-import sinon from 'sinon';
-import stu3FhirResources from '../testUtils/stu3FhirResources';
-import r4FhirResources from '../testUtils/r4FhirResources';
+delete r4DiagnosticReportExamples.DiagnosticreportExample;
 
-chai.use(sinonChai);
-
-const { expect } = chai;
+// includes 'Bundle'
+delete stu3QuestionnaireExamples.QuestionnaireProfileExampleUssgFht;
+// includes 'ReferralRequest' resource, not supported
```

Comment:
Keeping these comments makes this a bit odd, they were there to signal a resource's existence and explain why it not tested.

Reply:
But doesn't it still make sense now? 
Because the resources exist and are even in the Object. So the comment explains why they are deleted in the following line. It also makes sense for me to keep those resources in the folders, as they are of a supported type and show the shortcomings of the current implementation.

==> They explain the following line. Blumly, a young talented programmer that does not know the code, would otherwise wonder why we are randomly deleting test resources.

Reply:
But you don't see that right there. The original point had a comment next to a COMMENTED resource, so the next step is obviously "ah, that must be the reason why they don't use it, but good to know it's there", whereas now I have to know what specifically we're doing with this resource down the road.

Reply:
Clarified in Call: Agreed that comments stay